pyrene/src/components/DropdownButton/DropdownButton.tsx

```diff
@@ -55,7 +55,7 @@ const DropdownButton: FunctionComponent<DropdownButtonProps> = ({
     <Popover
       displayPopover={displayActions}
       onClickOutside={onClose}
-      renderPopoverContent={() => <OptionsList actions={actions} onClick={onClose} />}
+      renderPopoverContent={() => <OptionsList actions={actions || [{ label: 'tbd', onClick: () => null }]} onClick={onClose} />}
```

Comment:
Why do we need it here as well? If you don't pass actions, shouldn't there either be an empty dropdown, or an error?